src/provider/controller/ToastManager.ts

```diff
@@ -207,3 +206,7 @@ export class ToastManager extends AsyncInit {
         }));
     }
 }
+
+function retireToast(toast: Toast): void {
```

Comment:
This felt natural since it doesn't access any class members. And declaring it inside `onAction` would be pretty unusual for us. Let's have a 5 second battle of wills when you get in?

Reply:
✅